src/styles/blocks/cardBuy.scss

```diff
@@ -0,0 +1,24 @@
+.card__buy {
+  box-sizing: border-box;
+  height: 40px;
+  width: 166px;
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  background: $link-bg-color;
+  color: $link-text-color;
+  border-radius: 5px;
+  margin-top: 0;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 40px;
+  border: 1px solid $link-bg-color;
+
+  &:hover {
+    box-sizing: border-box;
+    background-color: $link-text-color;
+    color: $link-bg-color;
+    border: 1px solid $link-bg-color;
+    line-height: 39px;
```

Comment:
Changing the `line-height` on hover can lead to a change in the element's height which can cause a layout shift. It's better to use padding to adjust the button's appearance on hover while keeping the size consistent.